handlers/error.go

```diff
@@ -76,6 +79,14 @@ var (
 	ErrParseDuration    = errors.New(ErrParseDurationCode, errors.Alert, []string{"error parsing test duration"}, []string{}, []string{}, []string{"please refer to:  https://docs.meshery.io/guides/mesheryctl#performance-management"})
 )
 
+func ErrHealthCheckFailed(err error) error {
+	return errors.New(ErrHealthCheckFailedCode, errors.Alert, []string{"health checks failed"}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrInvalidAdapter(err error, obj string) error {
+	return errors.New(ErrInvalidAdapterCode, errors.Alert, []string{"invalid adapter " + obj + " specified"}, []string{err.Error()}, []string{}, []string{})
+}
```

Comment:
You can consider writing the short description like `return errors.New(ErrInvalidAdapterCode, errors.Alert, []string{"invalid adapter " , obj , " specified"}, []string{err.Error()}, []string{}, []string{})` We use "," rather using "+" while appending a string in this meshkit errors.